### PR TITLE
Add new EKS periodic job

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @gyuho

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -5728,12 +5730,15 @@ dashboards:
   dashboard_tab:
   - name: ci-kubernetes-e2e-stable-aws-eks-1-11-prod
     test_group_name: ci-kubernetes-e2e-stable-aws-eks-1-11-prod
-    description: Kubernetes stable branch e2e tests with EKS prod build 1.10
+    description: Kubernetes stable branch e2e tests with EKS prod build 1.11
```
Comment (BenTheElder) [inline testgrid/config.yaml]: uh, are we just testing EKS and not k/k?? I don't see any build or anything like that happening?

Comment (shyamjvs) [reply testgrid/config.yaml]: > are we just testing EKS and not k/k??

Not exactly. Yes we're testing against EKS, but EKS itself is based on the k/k release (which is being updated from time to time). That said, there is still part of the k/k build we're using for the test, like kubectl client code, e2e tests, etc..

File: config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml [hunk 2]
```diff
@@ -99,3 +99,27 @@ periodics:
...
+      args:
+      - --timeout=200
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-version-skew=false
+      - --deployment=eks
+      - --provider=eks
+      - --gce-ssh=
+      - --extract=ci/latest
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: `ci/latest` has a 1.14-alpha build, I assume this means we are NOT testing Kubernetes at all then ...

File: testgrid/config.yaml (hunk 1 above)
Comment (BenTheElder) [reply testgrid/config.yaml]: >  etc..

so just kubectl and the e2e tests? these are not even supported. 1.11 <--> 1.14 is not a supported matrix.

File: config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml (hunk 2 above)
Comment (shyamjvs) [reply config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: See my comment above.

One more thing - I think we should change this to ci/stable3 or something to make sure we're not running newer tests against older k/k release.

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: changing this to stable3 makes more sense.

File: testgrid/config.yaml (hunk 1 above)
Comment (shyamjvs) [reply testgrid/config.yaml]: Agree, and that's why https://github.com/kubernetes/test-infra/pull/10549#discussion_r244864693

File: config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml (hunk 2 above)
Comment (shyamjvs) [reply config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: Will change it soon.

**Pull request merged**